model/ftn/w3tidemd.ftn

```diff
@@ -463,7 +463,7 @@
 !
 ! Defines names and frequencies
 !
-      IF (ALLOCATED(TIDE_FREQC)) DEALLOCATE(TIDE_FREQC)
+      IF (ALLOCATED(TIDE_FREQC)) DEALLOCATE(TIDE_FREQC,TIDECON_NAME)
```

Comment:
Hi @SanderHulst 
I ran into a problem when running one of our regression test. The fix is if we revert this line. 
IF (ALLOCATED(TIDE_FREQC)) DEALLOCATE(TIDE_FREQC)
Could you confirm? 
The problematic test is ww3_tp2.18 (tide).
./bin/run_test -c intel -S -T -s TIDE   -w work_TIDE -o netcdf      ../model ww3_tp2.18

Reply:
Hi,
the required netcdf file is located on our ftp. Please use https://github.com/NOAA-EMC/WW3/blob/develop/model/bin/ww3_from_ftp.sh at the top of your directory, so it downloads the tar file, and copies the requires files for each regtest. Please try to reproduce the problem I mentioned. Or let me know if you could not grab them, so I can share them with you.

Reply:
Hi @SanderHulst again,
If you could not reproduce the bug, please merge noaa/develop into your PR, it might pop up.

Reply:
Fixed. I **assumed** that if TIDE_FREQC was allocated, TIDECON_NAME would be allocated as well as they operate in pairs. This was not the case.

Tidied up a bit and merged develop into this branch